radis/test/lbl/test_loader.py

```diff
@@ -86,9 +86,27 @@ def test_retrieve_from_database(
         rmtree(temp_database_name)
 
 
+def test_ignore_cached_files():
+
+    sf = SpectrumFactory(wavenum_min=2000, wavenum_max=3000, pressure=1,)
+
+    loaded_files = True
+    file_dir = getTestFile("cdsd_hitemp_09_fragment.txt")
+    test_file = file_dir[:-8] + "*"
+    sf.load_databank(path=test_file, format="cdsd-hitemp", parfuncfmt="hapi")
+
+    try:
+        sf.load_databank(path=test_file, format="cdsd-hitemp", parfuncfmt="hapi")
+    except UnicodeDecodeError:
+        loaded_files = False
+
+    assert loaded_files
```

Comment:
I would make a more explicit error  :

```suggestion
    try:
        sf.load_databank(path=test_file, format="cdsd-hitemp", parfuncfmt="hapi")
    except UnicodeDecodeError as err:
        raise UnicodeDecodeError("Couldnt load database the 2nd time. This may be due to cache files trying to be read as normal files") from err
```